scripts/run_all_tests.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
-# bash strict mode
-set -euo pipefail
```

Comment:
Did this cause issues?
It forces the bash script to fail if some component fails and that is generally the sane thing to do.
See [bash strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/).

Reply:
I removed it, so I can use `trap` command [here](https://github.com/lanterndata/lanterndb/pull/50/files/050358687cd6d80e233e34eb502f7c024f66f9df#diff-37a6113e1968944c1e8d34be76b1ce800af332c777659556edb55904af465122L69)